database/migrations/migrations.go

```diff
@@ -75,6 +78,22 @@ func (e EmbedFSWrapper) Open(name string) (fs.File, error) {
 
 	s := string(b)
 
+	privateKey, err := crypto.GenerateKey()
+	if err != nil {
+		return nil, err
+	}
+
+	publicKey := privateKey.Public()
+	publicKeyECDSA, ok := publicKey.(*ecdsa.PublicKey)
+	if !ok {
+		return nil, errors.New("Unable to parse public key")
+	}
+
+	publicKeyBytes := crypto.FromECDSAPub(publicKeyECDSA)
+	privateKeyBytes := crypto.FromECDSA(privateKey)
+
+	constants["NODE_KEY"] = hexutil.Encode(publicKeyBytes)[4:] + ":" + hexutil.Encode(privateKeyBytes)[2:]
```

Comment:
I would just do this as a struct with Public and Private fields that is JSON serializable. 
It is stored in attribute, so json(b) and both read and written only internally. So a simple struct can be used to bind (read&write) this attribute instead of custom formatting/splitting.